mix.exs

```diff
@@ -40,11 +40,12 @@ defmodule Membrane.FLV.Mixfile do
       {:membrane_core, "~> 0.11.2"},
       {:membrane_aac_format, "~> 0.7.0"},
       {:membrane_mp4_format, "~> 0.7.0"},
-      {:membrane_h264_format, "~> 0.4.0"},
+      {:membrane_h264_format, "~> 0.5.0"},
       {:membrane_file_plugin, "~> 0.13.2", only: :test},
-      {:membrane_h264_ffmpeg_plugin, "~> 0.25.1", only: :test},
+      {:membrane_h264_ffmpeg_plugin, "~> 0.25.3", only: :test},
       {:membrane_aac_plugin, "~> 0.13.0", only: :test},
-      {:membrane_mp4_plugin, "~> 0.18.0", only: :test},
+      {:membrane_mp4_plugin,
+       github: "membraneframework/membrane_mp4_plugin", branch: "bump-dependencies", only: :test},
```

Comment:
It's a test dependency, so we technically could merge it like that, but I'd rather not. Please change it to hex dependency before you merge